Infrastructure/DataImporterService.cs

```diff
@@ -191,32 +194,228 @@ private async Task RemoveGeneralInformationAboutSchools(NtpDbContext dbContext,
+            .NewConfig()
+            .Ignore(dest => dest.Id)
+            .Ignore(dest => dest.TuitionPartnerId)
+            .Ignore(dest => dest.TuitionPartner!)
+            .Ignore(dest => dest.TuitionType!)
+            .Ignore(dest => dest.LocalAuthorityDistrict!);
+
+        TypeAdapterConfig<SubjectCoverage, SubjectCoverage>
+            .NewConfig()
+            .Ignore(dest => dest.Id)
+            .Ignore(dest => dest.TuitionPartnerId)
+            .Ignore(dest => dest.TuitionPartner!)
+            .Ignore(dest => dest.TuitionType!)
+            .Ignore(dest => dest.Subject!);
+
+        TypeAdapterConfig<Price, Price>
+            .NewConfig()
+            .Ignore(dest => dest.Id)
+            .Ignore(dest => dest.TuitionPartnerId)
+            .Ignore(dest => dest.TuitionPartner!)
+            .Ignore(dest => dest.TuitionType!)
+            .Ignore(dest => dest.Subject!);
+
+        TypeAdapterConfig<TuitionPartner, TuitionPartner>
+            .NewConfig()
+            .Ignore(dest => dest.Id)
+            .Ignore(dest => dest.Prices!)
+            .Ignore(dest => dest.LocalAuthorityDistrictCoverage!)
+            .Ignore(dest => dest.SubjectCoverage!)
+            .Ignore(dest => dest.Logo!)
+            .Ignore(dest => dest.HasLogo)
+            .Ignore(dest => dest.ImportProcessLastUpdatedData)
+            .Ignore(dest => dest.IsActive)
+            .Ignore(dest => dest.OrganisationType);
+    }
+
+    private static void ImportTuitionPartnerLocalAuthorityDistrictCoverage(NtpDbContext dbContext, TuitionPartner existingTP, TuitionPartner tuitionPartnerToProcess)
+    {
+        var laDistrictCoveragesToDelete = existingTP.LocalAuthorityDistrictCoverage.Where(x => !tuitionPartnerToProcess.LocalAuthorityDistrictCoverage.Any(e => e.TuitionTypeId == x.TuitionTypeId &&
+                                                                                                                                                e.LocalAuthorityDistrictId == x.LocalAuthorityDistrictId));
+        foreach (var laDistrictCoverageToDelete in laDistrictCoveragesToDelete)
+        {
+            dbContext.Entry(laDistrictCoverageToDelete).State = EntityState.Deleted;
+        }
+
+        foreach (var laDistrictCoverage in tuitionPartnerToProcess.LocalAuthorityDistrictCoverage)
+        {
+            var existingLaDistrictCoverage = existingTP.LocalAuthorityDistrictCoverage.FirstOrDefault(x => x.TuitionTypeId == laDistrictCoverage.TuitionTypeId &&
+                                                                                                        x.LocalAuthorityDistrictId == laDistrictCoverage.LocalAuthorityDistrictId);
+
+            if (existingLaDistrictCoverage == null)
+            {
+                existingTP.LocalAuthorityDistrictCoverage.Add(laDistrictCoverage);
+            }
+            else
+            {
+                existingLaDistrictCoverage = laDistrictCoverage.Adapt(existingLaDistrictCoverage);
```

Comment:
The value assigned is not used in any execution path. Do we need it?

Reply:
This is used, because it updates the object (by ref) and this is then flagged as modified (if data has changed) with the following check.  It's then subsequently saved. 
                    dbContext.ChangeTracker.Entries().Any(e => e.State == EntityState.Modified ||
                                                                e.State == EntityState.Added ||
                                                                e.State == EntityState.Deleted

I have added a comment to explain this is the case.